### PR TITLE
Add nidmm and nidcpower system tests for attribute types not yet covered

### PR DESCRIPTION
**Pull request by injaleea**
[X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

~~[ ] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~~

~~[ ] I've added tests applicable for this pull request~~

### What does this Pull Request accomplish?
Adding more systemtests for nidcpower and nidmm to cover missing method calls.
Added tests,
test_set_get_vi_int_64_attribute for nidcpower and
test_get_vi_string_attribute for nidmm

### What testing has been done?

Systemtest, tox

### CODE REVIEW

**Comment by injaleea** [comment]
image

This will make nidcpower and nidmm to have 100% coverage on systemtests

File: src/nidmm/metadata/functions_addon.py [hunk 1]
```diff
@@ -50,6 +50,8 @@
     'ConfigureWaveformCoupling':       { 'codegen_method': 'no',       },
     'ConfigureADCCalibration':         { 'codegen_method': 'no',       },
     'revision_query':                  { 'codegen_method': 'no',       },
+    'GetAttributeViString':            { 'codegen_method': 'no',       },  # Not used in Python APIs
```
Comment (marcoskirsch) [inline src/nidmm/metadata/functions_addon.py]: You cannot do this! NI-DMM does have attributes of type string:
```
Kirsch0:nimi-python kirsch$ grep attributes\.AttributeViString generated/nidmm/session.py 
    driver_setup = attributes.AttributeViString(1050007)
    group_capabilities = attributes.AttributeViString(1050401)
    instrument_firmware_revision = attributes.AttributeViString(1050510)
    instrument_manufacturer = attributes.AttributeViString(1050511)
    instrument_model = attributes.AttributeViString(1050512)
    logical_name = attributes.AttributeViString(1050305)
    serial_number = attributes.AttributeViString(1150054)
    specific_driver_description = attributes.AttributeViString(1050514)
    specific_driver_vendor = attributes.AttributeViString(1050513)
    supported_instrument_models = attributes.AttributeViString(1050327)
```

File: src/nidcpower/metadata/functions_addon.py [hunk 2]
```diff
@@ -42,6 +42,8 @@
     'Disable.+Trigger':                { 'codegen_method': 'no',       },
     'revision_query':                  { 'codegen_method': 'no',       },
     'MeasureMultiple':                 { 'codegen_method': 'no',       },  # Issue 444
+    'GetAttributeViInt64':             { 'codegen_method': 'no',       },  # Not used in python API
```
Comment (marcoskirsch) [inline src/nidcpower/metadata/functions_addon.py]: You cannot do this! NI-DCPower does have one attribute of type ViInt64!

```
Kirsch0:nimi-python kirsch$ grep attributes\.AttributeViInt64 generated/nidcpower/session.py 
    active_advanced_sequence_step = attributes.AttributeViInt64(1150075)
```

**Comment by marcoskirsch** [comment]
It appears you have this wrong. NI-DMM does indeed lack attribute of type ViInt64 but that's it.
A better question is why your changes weren't caught by system_tests. Please look into what's missing there.

In the case of NI-DMM, we have read-only string attributes and none are writable. But I think that removing niDMM_SetAttributeVIString is the wrong thing to do because:

    1. It will be needed whenever NI-DMM adds a writable attribute
    2. The error that not having the method will be worse than the error returned by the driver "this attribute is not writable".

**Comment by marcoskirsch** [comment]
> Updating functions_addon for nidcpower and nidmm

On the title of the PR, I've noticed this trend. You should try and be descriptive. Sure you updated those files... but what is it you did, at a high-level, in a descriptive manner that conveys intent?

> Disabling codegen of unused methods Get/SetAttributeViString for nidmm and Get/SetAttributeViInt64 fro nidcpower.

This is just paraphrasing the diff. The interesting part is... why are you doing this?

**Comment by injaleea** [comment]
We have 100% coverage on nidmm and nidcpower with this PR.

image

**Pull request merged**